components/header-bar/src/online-status.js

```diff
@@ -18,15 +22,24 @@ const useOnlineStatusInfo = ({ online, lastOnline }) => {
             relativeTime: moment(lastOnline).fromNow(),
         })
     }
-    // todo: in the future, support 'CUSTOM' option
+
+    if (onlineStatusMessage) {
+        return onlineStatusMessage
+    }
 
     return null
 }
 
 /** A badge to display online/offline status in the header bar */
 export function OnlineStatus({ dense }) {
     const { online, lastOnline } = useOnlineStatus()
-    const info = useOnlineStatusInfo({ online, lastOnline })
+    const { onlineStatusMessage } = useOnlineStatusMessage()
+
+    const info = useOnlineStatusInfo({
+        online,
+        lastOnline,
+        onlineStatusMessage,
+    })
```

Comment:
Following from the above, basically this would become: 
```suggestion
    const info = onlineStatusMessage
```
... which basically makes the info variable unnecessary

Reply:
Yep totally :-)

Reply:
Directly using `onlineStatusMessage` now